schema/review-json-schema.md

```
@@ -0,0 +1,535 @@
+  ],
+  "violations": [],
+  ...
+}
+$ echo $?
+0
+```
+
+`violations: []`, exit `0`. The gate passed because it was never given anything to check, and that
+outcome is byte-identical to a genuine pass everywhere a caller looks (exit code, `violations`,
+`cause`). The two knobs that produce it are [`round`](#round) and
+[`rounds_audit[].strike`](#roundsauditstrike) — read those two sections before writing a verdict by
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)